bokeh/plotting/helpers.py

```diff
@@ -68,6 +67,110 @@ def _stack(stackers, spec0, spec1, **kw):
 
     return _kw
 
+def _graph(node_source, edge_source, **kwargs):
```

Comment:
This could be DRY-ed up a bit, but it's only repeated once so the benefit isn't huge.